### ISSUE TITLE
Mac doesn't run your code

### ISSUE THREAD

**Issue by iocohen** [andela-foladeji/License-Plate-Recognition-Nigerian-vehicles#29]
Hi there,

I've install all of your requirements, but still doesn't run the tool. do you know why?

**Comment by andela-foladeji** [comment]
Are you getting any error @iocohen

**Comment by iocohen** [comment]
Nope. I run the python3 Alr.py and is not appearing. Simply in terminal i see the code running and that is it. When i close the python app i see 15.

**Comment by iocohen** [comment]
i tried now with python 2 to run the code. it still doesnt say anything
